app/uk/gov/hmrc/excisemovementcontrolsystemapi/controllers/GetMovementsController.scala

```diff
@@ -111,10 +111,11 @@ class GetMovementsController @Inject()(
   }
 
   private def createResponseFrom(movement: Movement) = {
-    GetMovementResponse(
+    ExciseMovementResponse(
       movement._id,
-      movement.consignorId,
+      None,
```

Comment:
could we return the boxId here too? this would just be `movement.boxId`

Reply:
Main reason I didn't was because it only applies to the consignor and the consignee might be calling this so it seems like it might confuse them

Reply:
I'm not sure what's best here then as the boxId relates to a piece of software not the consignor. If a consignee needs to also get notified of new messages how do they do that without the boxId. They would need the movementId anyway to poll our API for new messages. It might be good to display the boxId in the get movements/get movement endpoints so we always show the relevant information for that movement. Could this potentially be a problem though if someone gets hold of a boxId not meant for the piece of software they are using??

Reply:
Maybe just leave as it is for now. I'm not sure what's best to do and if a third party questions it, we can maybe make a decision then

Reply:
Yeah I don't know if having the box Id for something you can't access is a problem or not. Without the callback url I assume its useless? I agree with like waiting and seeing what people say :)